RELEASE_NOTES.md

```diff
@@ -1,5 +1,39 @@
 # Release Notes
 
+
+## 2.0.0 Release Notes Summary
+
+A summary of changes in the 2.0.0 version of dxCompiler comparing to the previous dxWDL version 1:
```

Comment:
Is this compared against the latest version of dxWDL (v1.48.2) ?

Reply:
It is compared against v1.47.2, i.e. the version after which the first 2.0.0-rc was released, not against the latest dxWDL which was v1.50 (Dec 9th).

I compared against v1.47.2 to include **all** the changes that went into the `2.0.0-rc*` versions.